src/blog/lagom-1-5-0-M1.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Lagom 1.5.0-M1 released!
+date: 2018-07-25
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.5.0-M1
+---
+
+We are happy to announce the availability of Lagom 1.5.0-M1. This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and starts moving towards supporting scala 2.13.
+
+The motivation behind moving Lagom closer to Play and Akka is twofold: Play and Akka users will be able to adopt Lagom more easily by reusing existing code or APIs they are already familiar with, and framework maintainers will reduce the amount of code to maintain and converge in conventions speeding up the development process.
+
+## Highlights
+
+- Updates Play to [2.7.0-M2](https://blog.playframework.com/play-2-7-0-m2-released/) (see also [2.7.0-M1](https://blog.playframework.com/play-2-7-0-milestone-1-released/)). Play 2.7.x improves failure handling in RDBMS failure scenarios and adopts [Akka's Coordinated Shutdown](https://doc.akka.io/docs/akka/current/actors.html#coordinated-shutdown). It also removes some deprecated APIs Lagom exposed transparently so you may need to review your code. Make sure to review upstream changes. As part of this update, the dependency to the Jackson library (for java JSON serialization) was bumped to version `2.9.6` which introduced some new default settings that may require you to review your serialization settings.
+- Start adoption of [Akka's Service Discovery](https://developer.lightbend.com/docs/akka-management/current/discovery.html) which will become a building block of Lagom's `ServiceLocator` and help standardize service discovery scenarios. This will simplify production settings to tune cluster bootstrap and other service discovery strategies and combine them.
```

Comment:
I think we can be more specific here. (What does "start adoption" mean? Why does it matter?)

How about something like this?

Development mode support for [Akka Service Discovery]((https://developer.lightbend.com/docs/akka-management/current/discovery.html)). This new library provides an extensible API for looking up service addresses by name from within Akka-based services, including integration with Kubernetes, Marathon, Consul, and others. It is similar to Lagom's `ServiceLocator` API, but can be used in any Akka or Play application, as well as in Lagom. It is a foundational building block for many of the upcoming cross-product features, including gRPC support, and with this release it works seamlessly in Lagom's `runAll` development mode without additional configuration. We are planning to integrate this more fully with Lagom in future 1.5.0 milestones and beyond.